### PR TITLE
Add Ansible-express

### PR DESCRIPTION
**Pull request by peterMuriuki**
- Role to install express

### CODE REVIEW

File: molecule/default/converge.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    - name: "Include ansible-express"
+      include_role:
+        name: "ansible-express"
+      vars:
+        express_system_user: "express"
+        express_system_group: "www-data"
+        express_system_user_home: "/home/{{ express_system_user }}"
+        express_node_version: 10.x
+        express_git_url: "https://github.com/onaio/reveal-frontend"
+        express_git_version: "tiny-backend"
```
Comment (moshthepitt) [inline molecule/default/converge.yml]: This branch on the Reveal repo was deleted

File: tasks/cleanup.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+- name: Get previous app deploy listing
+  command: ls -tr1 chdir={{ express_versioned_path }}
+  register: versioned_list
+  changed_when: false
+
+- name: Remove extra files from app directory
+  file:
+    path: "{{ express_versioned_path }}/{{ versioned_list.stdout_lines[item|int] }}"
+    state: absent
+  with_sequence: start=0 end={{ versioned_list.stdout_lines|length - 10 }}
```
Comment (moshthepitt) [inline tasks/cleanup.yml]: Lets make `10` a variable.

**Review by moshthepitt — CHANGES_REQUESTED** [review]

File: molecule/default/converge.yml (hunk 1 above)
Comment (peterMuriuki) [reply molecule/default/converge.yml]: will change that to master.

File: tasks/cleanup.yml (hunk 2 above)
Comment (peterMuriuki) [reply tasks/cleanup.yml]: ok

File: molecule/default/converge.yml [hunk 3]
```diff
@@ -11,6 +11,6 @@
         express_system_user_home: "/home/{{ express_system_user }}"
         express_node_version: 10.x
         express_git_url: "https://github.com/onaio/reveal-frontend"
-        express_git_version: "tiny-backend"
+        express_git_version: "master"
```
Comment (moshthepitt) [inline molecule/default/converge.yml]: I think that it is better to use a specific commit hash

Comment (peterMuriuki) [reply molecule/default/converge.yml]: ok

Comment (moshthepitt) [reply molecule/default/converge.yml]: Or a release tag - could also work

Comment (peterMuriuki) [reply molecule/default/converge.yml]: Yeah, i will go with the release tag, I agree its actually better.

**Review by morrismukiri — APPROVED** [review]

**Review by moshthepitt — APPROVED** [review]

**Pull request merged**